app/views/research_sessions/preview.html.erb

```diff
@@ -157,18 +136,112 @@
           <% end %>
         </p>
       <% end %>
-    </section>
-  <% end %>
-  <% if @research_session.incentives_enabled %>
-    <section>
-      <h3 class="subtitle-small">Incentives</h3>
-      <p>
-        We are offering
-        <%= edit_link_for(:incentive_value) do %><%= @research_session.incentive_text %><% end %>
-        for participation in this session.
-      </p>
-    </section>
-  <% end %>
+    <% end %>
+  </section>
+
+  <section>
+    <h3 class="subtitle-small" id="confidential">
+      Will anyone know what <%= i_or_they %> say in the discussion?
+    </h3>
+
+    <p>
+      The only people who will hear what <%= you_or_your_child %> <%= say_or_says %> in the session
+      will be the researchers running the session, other young people taking part, and
+      service workers facilitating the session.
+    </p>
+
+    <p>
+      <%= edit_link_for(:shared_with) do %>
+        <%= shared_with_lookup(@research_session.shared_with) %>
+      <% end %>
+    </p>
+
+    <p>
+      Barnardo’s will hold research data for <%= edit_link_for(:shared_duration) %>
+      after the closure of this project, after which it will be deleted.
+      Personal data is stored in a safe and secure way.
+    </p>
+    <p>
+      You can contact Jason Caplin to ask us to delete your personal data at any time.
+    </p>
+  </section>
+
+  <section>
+    <h3 class="subtitle-small" id="sharing-information">Sharing information</h3>
+    <p>
+      We may have to share information about <%= you_or_your_child %> without
+      your consent if we think <%= you_or_they %> or someone else is at risk
+      of serious harm. Where possible, the lead researcher speak to you first
```

Comment:
"the lead researcher speak" should be changed to "the lead researcher will speak"